controller/csi_general/replication_pb2_grpc.py

```diff
@@ -0,0 +1,116 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
will it bee part of CSI-1377?

Reply:
yes, all `.proto` files should be handled in the same manner